docker-compose.debug.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.4'

```

Comment:
This is enough for debug compose file:

```
version: '3.7'

services:
  todo-api:
    command: nodemon --inspect-brk=0.0.0.0 .
    ports:
      - 9229:9229
```

Reply:
e34ccd39407d34189ac7db0e271bda287ee63182
done